modules/rostests/apitests/win32nt/ntuser/NtUserConvertMemHandle.c

```diff
@@ -0,0 +1,17 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for NtUserConvertMemHandle
+ * COPYRIGHT:   Copyright 2025 Max Korostil (mrmks04@yandex.ru)
+ */
+
+#include "../win32nt.h"
+
+
+START_TEST(NtUserConvertMemHandle)
+{
+    HANDLE hMem;
+ 
+    hMem = NtUserConvertMemHandle((PVOID)0xDEADBEEF, 0xFFFF);
+    TEST(hMem == NULL);
```

Comment:
First, set and check added "LastError".
Then,
```suggestion
    ok_hdl(hMem, NULL);
```